src/v2/view-builder/views/ov/ChallengeOktaVerifyFastPassView.js

```diff
@@ -40,7 +39,7 @@ const Body = BaseForm.extend(Object.assign(
     },
 
     remove() {
-      BaseForm.prototype.remove.apply(this, arguments);
+      BaseFormWithPolling.prototype.remove.apply(this, arguments);
```

Comment:
nit : Wondering if we can move the remove to BaseFormWithPolling as well ?

Reply:
That's the ultimate goal. Currently the `DeviceChallengePollView` and `ChallengeOktaVerifyFastPassView` are mostly the same with small diff such as the `remove` function. I am going to make incremental changes and eventually have them share the common part.